extensions/rust/src/route_map.rs

```diff
@@ -290,14 +292,12 @@ impl<'rm> ConfigureNodeView<'rm> {
 
         let asgi_handlers = cur_node.asgi_handlers.as_mut().unwrap();
 
-        macro_rules! generate_single_route_handler_stack {
-            ($handler_type:expr) => {
-                let route_handler = route.getattr("route_handler")?;
-                let middleware_stack =
-                    build_route_middleware_stack(py, &ctx, route, route_handler)?;
-                asgi_handlers.insert($handler_type.to_string(), middleware_stack.to_object(py));
-            };
-        }
+        let mut generate_single_route_handler_stack = |handler_type: &str| -> PyResult<()> {
+            let route_handler = route.getattr("route_handler")?;
+            let middleware_stack = build_route_middleware_stack(py, &ctx, route, route_handler)?;
+            asgi_handlers.insert(handler_type.to_string(), middleware_stack.into());
```

Comment:
I've just discovered that the `.into()` here is not necessary. PyObject is just an alias for Py<PyAny>, which is returned by `build_route_middleware_stack`. This also means that the call to `middleware_stack.to_object(py)` that I wrote before is also unnecessary, that one can be changed too